src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -19,6 +19,9 @@ use self::impl_op::*;
 use crate::coprocessor::codec::data_type::*;
 use crate::coprocessor::Result;
 
+/// Parses the signed / unsigned int in children, and returns the proper Comparer for the children.
```

Comment:
returns the corresponding comparing function.

Also you need to explain why should we do this, instead of using a single function for these (it's what we did previously) and checking signed / unsigned in the function implementation.